### ISSUE TITLE
TypeError: Illegal constructor in Safari on desktop

### ISSUE THREAD

**Issue by ClientsideDesign** [iVis-at-Bilkent/cytoscape.js-context-menus#55]
In Safari on desktop an `Illegal constructor` error is thrown, which prevents the menu from working:

<img width="535" alt="Screenshot 2020-11-24 at 18 01 02" src="https://user-images.githubusercontent.com/6104494/100133763-156b2600-2e7f-11eb-9bba-30c79ea78eb8.png">

All of your demos seem to be affected - happy to help with any further information.

Thanks

**Comment by msalihaltun** [comment]
I think this issue is caused by the usage of [custom elements](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements). I'm not sure we can polyfill it in a simple way, maybe we can use [this polyfill](https://github.com/ungap/custom-elements-builtin) but we need to be aware of this [section](https://github.com/ungap/custom-elements-builtin#constructor-caveat) in their README as we are using constructors (this is the reason it might be complicated to go for a polyfill solution?).

**Comment by lbakke** [comment]
I'm also getting this issue in Electron.

**Comment by ugurdogrusoz** [comment]
@ClientsideDesign @lbakke unfortunately we don't have resources to work on this at the moment. Feel free to make a patch and issue a PR on the *unstable* branch if you like.

**Comment by lbakke** [comment]
ok, no problem. @ugurdogrusoz  I see that you're also involved with development for cytoscape-edge-editing, could you tell me if you can use edge-editing in any extent without the use of context-menu's in the cytoscape graph?

**Comment by ugurdogrusoz** [comment]
@lbakke in fact none of our extensions depend on each other and all should be compatible with our knowledge. Enjoy! :)

**Comment by lbakke** [comment]
Okay, thank you! However when I try using edge-editing with the default parameters, right-clicking on an edge produces the error "contextMenus is not a function" which is why I thought it context-menu was necessary in order to use edge-editing. Is there a way to add bend points or remove them without using context-menu?

**Comment by ugurdogrusoz** [comment]
You are right I completely forgot that we need the context menus to facilitate adding and removing bend / control points.

**Comment by msalihaltun** [comment]
Can confirm that this issue is still persistent with latest version of Safari.

**Comment by matthughes** [comment]
And does anyone have a good way to check for this support so we can at least show the graph without the context menus.  Otherwise the whole page blows up.

**Comment by ClientsideDesign** [comment]
@matthughes - did you see the polyfill suggestion on here? Not tried it myself. https://github.com/danieldietrich/candid/issues/17#issuecomment-1028455087

**Comment by sveinp** [comment]
The suggestion seems to work in our case at least.

**Comment by JoshuaeKaiser** [comment]
This 

> @matthughes - did you see the polyfill suggestion on here? Not tried it myself. [danieldietrich/candid#17 (comment)](https://github.com/danieldietrich/candid/issues/17#issuecomment-1028455087)

This worked for me also. Not ideal, but atleast its a workaround.
